### PR TITLE
remove JPA properties that do not alter defaults

### PR DESCRIPTION
**Pull request by biergit**

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/generator/server/springboot/database/sqlcommon/domain/SQLCommonModuleBuilder.java [hunk 1]
```diff
@@ -1,6 +1,15 @@
 package tech.jhipster.lite.generator.server.springboot.database.sqlcommon.domain;
 
-import static tech.jhipster.lite.module.domain.JHipsterModule.*;
+import static tech.jhipster.lite.module.domain.JHipsterModule.JHipsterModuleBuilder;
```
Comment (DamnClin) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/database/sqlcommon/domain/SQLCommonModuleBuilder.java:3]: Can you set your static import limit to 1 in your IDE and reorganize imports in this file?

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/mariadb/domain/MariaDBModuleFactoryTest.java [hunk 2]
```diff
@@ -1,7 +1,8 @@
 package tech.jhipster.lite.generator.server.springboot.database.mariadb.domain;
 
 import static org.mockito.Mockito.when;
-import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.*;
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.assertThatModuleWithFiles;
```
Comment (DamnClin) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/database/mariadb/domain/MariaDBModuleFactoryTest.java:4]: Can you set your static import limit to 1 in your IDE and reorganize imports in this file?

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/mssql/domain/MsSQLModuleFactoryTest.java [hunk 3]
```diff
@@ -1,7 +1,10 @@
 package tech.jhipster.lite.generator.server.springboot.database.mssql.domain;
 
 import static org.mockito.Mockito.when;
-import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.*;
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.ModuleFile;
```
Comment (DamnClin) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/database/mssql/domain/MsSQLModuleFactoryTest.java:4]: Can you set your static import limit to 1 in your IDE and reorganize imports in this file?

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/mysql/domain/MySQLModuleFactoryTest.java [hunk 4]
```diff
@@ -1,7 +1,8 @@
 package tech.jhipster.lite.generator.server.springboot.database.mysql.domain;
 
 import static org.mockito.Mockito.when;
-import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.*;
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.assertThatModuleWithFiles;
```
Comment (DamnClin) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/database/mysql/domain/MySQLModuleFactoryTest.java:4]: Can you set your static import limit to 1 in your IDE and reorganize imports in this file?

File: src/test/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactoryTest.java [hunk 5]
```diff
@@ -1,7 +1,8 @@
 package tech.jhipster.lite.generator.server.springboot.database.postgresql.domain;
 
 import static org.mockito.Mockito.when;
-import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.*;
+import static tech.jhipster.lite.module.infrastructure.secondary.JHipsterModulesAssertions.assertThatModuleWithFiles;
```
Comment (DamnClin) [inline src/test/java/tech/jhipster/lite/generator/server/springboot/database/postgresql/domain/PostgresqlModuleFactoryTest.java:4]: Can you set your static import limit to 1 in your IDE and reorganize imports in this file?

File: src/main/java/tech/jhipster/lite/generator/server/springboot/database/sqlcommon/domain/SQLCommonModuleBuilder.java [hunk 6]
```diff
@@ -78,11 +87,8 @@ public static JHipsterModuleBuilder sqlCommonModuleBuilder(
           propertyValue("org.hibernate.boot.model.naming.CamelCaseToUnderscoresNamingStrategy")
         )
         .set(propertyKey("spring.jpa.open-in-view"), FALSE)
-        .set(propertyKey("spring.jpa.properties.hibernate.cache.use_second_level_cache"), FALSE)
```
Comment (DamnClin) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/database/sqlcommon/domain/SQLCommonModuleBuilder.java:81]: I have no advice on this :D. cc @DanielFran

Comment (pascalgrimaud) [reply src/main/java/tech/jhipster/lite/generator/server/springboot/database/sqlcommon/domain/SQLCommonModuleBuilder.java:81]: yes indeed, this property should be removed I think. It should be added only with the hibernate cache module

File: src/main/java/tech/jhipster/lite/generator/server/springboot/database/sqlcommon/domain/SQLCommonModuleBuilder.java [hunk 7]
```diff
@@ -2,18 +2,15 @@
...
-import tech.jhipster.lite.module.domain.LogLevel;
-import tech.jhipster.lite.module.domain.docker.DockerImageVersion;
-import tech.jhipster.lite.module.domain.file.JHipsterDestination;
-import tech.jhipster.lite.module.domain.file.JHipsterSource;
-import tech.jhipster.lite.module.domain.javabuild.ArtifactId;
-import tech.jhipster.lite.module.domain.javadependency.JavaDependency;
-import tech.jhipster.lite.module.domain.javadependency.JavaDependencyScope;
-import tech.jhipster.lite.module.domain.javaproperties.PropertyValue;
-import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+import tech.jhipster.lite.error.domain.*;
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/database/sqlcommon/domain/SQLCommonModuleBuilder.java:5]: @biergit : I think @DamnClin wanted you to configure your IDE about import only on static

Comment (biergit) [reply src/main/java/tech/jhipster/lite/generator/server/springboot/database/sqlcommon/domain/SQLCommonModuleBuilder.java:5]: Haha, ok, I'll try again.

**Comment by pascalgrimaud** [comment]
Thanks for your work here @biergit :)

**Comment by pascalgrimaud** [comment]
after discussing with @DamnClin, I'm adding a bounty on this and for the work done on other PRs which clean the generated application with SQL

Plz claim it @biergit 
Have a look at https://www.jhipster.tech/bug-bounties/ ;-)

**Comment by biergit** [comment]
> after discussing with @DamnClin, I'm adding a bounty on this and for the work done on other PRs which clean the generated application with SQL
> 
> Plz claim it @biergit Have a look at https://www.jhipster.tech/bug-bounties/ ;-)

Wow that's quite generous. Thanks so much I didn't expect it for the small improvements I've  done :)

**Comment by pascalgrimaud** [comment]
@biergit : plz post your open collective link here, so I can approved :-)

**Comment by biergit** [comment]
Ah thank you @pascalgrimaud . Here you go: https://opencollective.com/generator-jhipster/expenses/104519

**Comment by pascalgrimaud** [comment]
@biergit : approved :-)

**Pull request merged**